ui/tests/acceptance/client-current-test.js

```diff
@@ -121,27 +100,6 @@ module('Acceptance | clients current', function (hooks) {
     assert.dom('[data-test-chart-container="new-clients"]').doesNotExist();
     assert.dom('[data-test-chart-container="new-clients"]').doesNotExist();
 
-    // check chart displays correct elements and values
-    for (const key in CHART_ELEMENTS) {
-      const { mounts } = by_namespace[0];
-      let mountNumber = mounts.length < 10 ? mounts.length : 10;
-      let group = find(CHART_ELEMENTS[key]);
-      let elementArray = Array.from(group.children);
-      assert.equal(elementArray.length, mountNumber, `renders correct number of ${key}`);
-      if (key === 'totalValues') {
-        elementArray.forEach((element, i) => {
-          assert.equal(element.innerHTML, `${mounts[i].counts.clients}`, 'displays correct value');
-        });
-      }
-      if (key === 'yLabels') {
-        elementArray.forEach((element, i) => {
-          assert
-            .dom(element.children[1])
-            .hasTextContaining(`${mounts[i].mount_path}`, 'displays correct auth label');
-        });
-      }
-    }
-
```

Comment:
I'm guessing the flaky failures came from within these loops? Is there a plan to revisit this and add back in more stable coverage for checking the chart elements and values?

Reply:
The tests were flaky prior, but yes - the tests failing after removing the original failing assertions definitely indicates that the loops were slowing things down. 

Definitely want to revisit - but will have to brainstorm the best way to approach. I'm thinking of moving it to the component test, rather than have it be incorporated in the acceptance.

Reply:
Nice! Sounds good to me.